ckcp/openshift_dev_setup.sh

```diff
@@ -361,6 +386,10 @@ install_pipeline_service() {
     WEBHOOK_SECRET="placeholder_webhook" \
     "$GITOPS_DIR/pac/setup.sh" | indent 4
 
+  echo -n "- Install tekton-results DB: "
+  kubectl apply -k "manifests/tekton-results-db/openshift" 2>&1 |
```

Comment:
You will need to prefix `$CKCP_DIR` otherwise this is not going to work if run outside of ckcp folder

Reply:
updated now